set_version.sh

```diff
@@ -0,0 +1,20 @@
+#!bin/bash
+#*****************************************************************************
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+first="$1"
+shift
+rest="${@}"
+sed -i -e 's|'$first'|'"$rest"'|g' /ovms/src/version.hpp
```

Comment:
do we need this script at all while the dockerfile is using sed command anyway. Maybe the dockerfile should edit the version.hpp directly. It should be simpler.